packages/monitors/src/SyntheticPegMonitor.js

```diff
@@ -180,14 +180,16 @@ class SyntheticPegMonitor {
     if (this.pegVolatilityAlertThreshold === 0) return; // Exit early if not monitoring peg volatility.
     const pricefeed = this.medianizerPriceFeed;
 
-    const volData = await this._checkPricefeedVolatility(pricefeed);
-
-    if (Object.keys(volData).includes("errorData")) {
+    // _checkPricefeedVolatility either returns successfully or throws
+    let volData;
+    try {
+      volData = await this._checkPricefeedVolatility(pricefeed);
```

Comment:
Simplify `_checkPricefeedVolatility` to either return successfully or throw, and catch that error here.